keras_cv/layers/object_detection/nms_prediction_decoder.py

```diff
@@ -19,7 +19,7 @@
 
 # TODO(tanzhenyu): provide a TPU compatible NMS decoder.
 @tf.keras.utils.register_keras_serializable(package="keras_cv")
-class NmsDecoder(tf.keras.layers.Layer):
+class NonMaxSuppression(tf.keras.layers.Layer):
```

Comment:
Let's also rename this file (and corresponding test file) to match this

Reply:
Done